docs/migration/guide_aspnet.md

```diff
@@ -0,0 +1,102 @@
+
+## Steps to Migrate
+
+### Step 1: Remove Application Insights SDK
+
+When you first added ApplicationInsights to your project, the SDK would have added a config file and made some edits to the web.config.
+If using Nuget tools to remove the Application Insights, some of this will be cleaned up. 
+If you're manually removing the package reference from your csproj, you'll need to manually cleanup these artifacts.
+
+- Remove any Microsoft.ApplicationInsights.* packages from you csproj and packages.config.
+- Delete the ApplicationInsights.config file.
+- Clean up your application's Web.Config file.
+
+    The following sections would have been automatically added to your web.config when you first added ApplicationInsights to your project. References to the `TelemetryCorrelationHttpModule` and the `ApplicationInsightsWebTracking` should be removed.
+
+    ```xml
+    <configuration>
+      <system.web>
+        <httpModules>
+          <add name="TelemetryCorrelationHttpModule" type="Microsoft.AspNet.TelemetryCorrelation.TelemetryCorrelationHttpModule, Microsoft.AspNet.TelemetryCorrelation" />
+          <add name="ApplicationInsightsWebTracking" type="Microsoft.ApplicationInsights.Web.ApplicationInsightsHttpModule, Microsoft.AI.Web" />
+        </httpModules>
+      </system.web>
+      <system.webServer>
+        <modules>
+          <remove name="TelemetryCorrelationHttpModule" />
+          <add name="TelemetryCorrelationHttpModule" type="Microsoft.AspNet.TelemetryCorrelation.TelemetryCorrelationHttpModule, Microsoft.AspNet.TelemetryCorrelation" preCondition="managedHandler" />
+          <remove name="ApplicationInsightsWebTracking" />
+          <add name="ApplicationInsightsWebTracking" type="Microsoft.ApplicationInsights.Web.ApplicationInsightsHttpModule, Microsoft.AI.Web" preCondition="managedHandler" />
+        </modules>
+      </system.webServer>
+    </configuration>
+    ```
+
+### Step 1: Install the OpenTelemetry SDK and Enable at Application Startup
+
+The OpenTelemery SDK must be configured at application startup. This is typically done in the Global.asax.cs.
+OpenTelemetry has a concept of three signals; Traces (Requests and Dependencies), Metrics, and Logs.
+Each of these signals will need to be configured as part of your application startup.
+
+#### Global.asax.cs
+
+TODO: NEED A SAMPLE THAT INCLUDES LOGGING.
+
+```csharp
+using OpenTelemetry;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Trace;
+
+public class MvcApplication : System.Web.HttpApplication
+{
+    private TracerProvider? tracerProvider;
+    private MeterProvider? meterProvider;
+
+    protected void Application_Start()
+    {
+        this.tracerProvider = Sdk.CreateTracerProviderBuilder()
+            .Build();
+
+        this.meterProvider = Sdk.CreateMeterProviderBuilder()
+            .Build();
+    }
+
+    protected void Application_End()
+    {
+        this.tracerProvider?.Dispose();
+        this.meterProvider?.Dispose();
+    }
+}
+```
```

Comment:
i'd recommend what happens at the end of this step - in this case nothing happens, but make it very clear that this step is just a prep-step only, and no telemetry will be collected at this point.